examples/dom_test/loopcond.out

```diff
@@ -0,0 +1 @@
+{'entry': {'entry'}, 'loop': {'loop', 'entry'}, 'exit': {'loop', 'exit', 'entry'}, 'body': {'loop', 'entry', 'body'}, 'then': {'then', 'loop', 'entry', 'body'}, 'endif': {'loop', 'endif', 'entry', 'body'}}
```

Comment:
The bummer about this one is that as implemented, `dom.py` doesn't produce a deterministic list. Our project 2 enforces CFG order on it with a somewhat hacky solution; but short of that the main Bril repo could also just sort based on something arbitrary like the block name.

Reply:
Good point! Maybe that command should produce an OrderedDict or something… but in any case, seems fine for now!